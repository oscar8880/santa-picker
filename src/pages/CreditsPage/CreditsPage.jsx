import styles from './CreditsPage.module.scss';
import React from 'react';
import Text from '../../components/Text/Text';
import Action from '../../components/Action/Action';
import PageWrapper from '../../components/PageWrapper/PageWrapper';
import { Container, Row, Col } from 'react-grid-system';

const CreditsPage = () => {
  return (
    <div className={styles.CreditsPage}>
      <PageWrapper>
        <Container fluid>
          <Row>
            <Col xs={12} md={8} lg={8}>
              <div className={styles.CreditsPage__Title}>
                <Text appearance="h6" tagName="h1">
                  Credits
                </Text>
              </div>
              <ul className={styles.CreditsPage__CreditList}>
                <li className={styles.CreditsPage__CreditListItem}>
                  <Text>
                    Header/footer background ‘Argyle’ made by Will Monson from:
                    <Action
                      href="https://www.flaticon.com/authors/pixelmeetup"
                      title="Pixelmeetup"
                      appearance="a"
                    >
                      https://www.transparenttextures.com
                    </Action>
                  </Text>
                </li>
                <li className={styles.CreditsPage__CreditListItem}>
                  <Text>
                    Icons made by{' '}
                    <Action
                      href="https://www.flaticon.com/authors/pixelmeetup"
                      title="Pixelmeetup"
                      appearance="a"
                    >
                      Pixelmeetup
                    </Action>
                    ,{' '}
                    <Action
                      href="http://www.freepik.com/"
                      title="Freepik"
                      appearance="a"
                    >
                      Freepik
                    </Action>{' '}
                    and{' '}
                    <Action
                      href="https://www.flaticon.com/authors/pixel-perfect"
                      title="Pixel perfect"
                      appearance="a"
                    >
                      Pixel Perfect
                    </Action>{' '}
                    from:{' '}
                    <a href="https://www.flaticon.com/" title="Flaticon">
                      www.flaticon.com
                    </a>
                  </Text>
                </li>
                <li className={styles.CreditsPage__CreditListItem}>
                  <Text></Text>
                </li>
              </ul>
              <div className={styles.CreditsPage__CTA}>
                <Action tagName="Link" to="/" appearance="button-secondary">
                  Home
                </Action>
              </div>
            </Col>
          </Row>
        </Container>
      </PageWrapper>
    </div>
  );
};

export default CreditsPage;
