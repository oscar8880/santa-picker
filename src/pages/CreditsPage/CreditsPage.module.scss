@import "../../scss/functions";
@import "../../scss/variables";

.CreditsPage {
  display: flex;
  flex-direction: column;
  padding-top: $spacer-10;
  @media (min-width: $breakpoint-md) {
    padding-bottom: $spacer-12;
    padding-top: $spacer-12;
  }
}

.CreditsPage__Title {
  padding-bottom: $spacer-8;
  text-align: center;
  @media (min-width: $breakpoint-md) {
    text-align: left;
  }
}

.CreditsPage__CreditList {
  list-style: none;
  padding-bottom: $spacer-4;
  text-align: center;
  @media (min-width: $breakpoint-md) {
    text-align: left;
  }
}

.CreditsPage__CreditListItem {
  color: $grey-500;
  padding-bottom: $spacer-6;
}

.CreditsPage__CTA {
  display: flex;
  justify-content: center;
  padding-bottom: $spacer-8;

  @media (min-width: $breakpoint-md) {
    justify-content: flex-start;
  }
}
