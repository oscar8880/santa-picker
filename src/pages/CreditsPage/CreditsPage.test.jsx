import React from 'react';
import { render } from '@testing-library/react';

import CreditsPage from './CreditsPage';

test('matches snapshot', async () => {
  const { asFragment } = render(<CreditsPage />);

  expect(asFragment()).toMatchSnapshot();
});
