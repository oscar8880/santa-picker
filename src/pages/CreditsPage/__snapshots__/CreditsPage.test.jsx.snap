// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="CreditsPage"
  >
    <div
      class="PageWrapper"
    >
      <div
        style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px;"
      >
        <div
          style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
        >
          <div
            style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 58.333333333333336%; flex-basis: 58.333333333333336%; flex-grow: 0; flex-shrink: 0; max-width: 58.333333333333336%; margin-left: 0%;"
          >
            <div
              class="CreditsPage__Title"
            >
              <h1
                class="Text Text__h1"
              >
                Credits
              </h1>
            </div>
            <ul
              class="CreditsPage__CreditList"
            >
              <li
                class="CreditsPage__CreditListItem"
              >
                <p
                  class="Text Text__p"
                >
                  Icons made by 
                  <button
                    class="Action Action__Appearance--a"
                    href="https://www.flaticon.com/authors/pixelmeetup"
                    title="Pixelmeetup"
                  >
                    Pixelmeetup
                  </button>
                  , 
                  <button
                    class="Action Action__Appearance--a"
                    href="http://www.freepik.com/"
                    title="Freepik"
                  >
                    Freepik
                  </button>
                   and 
                  <button
                    class="Action Action__Appearance--a"
                    href="https://www.flaticon.com/authors/pixel-perfect"
                    title="Pixel perfect"
                  >
                    Pixel Perfect
                  </button>
                   from: 
                  <a
                    href="https://www.flaticon.com/"
                    title="Flaticon"
                  >
                    www.flaticon.com
                  </a>
                </p>
              </li>
              <li
                class="CreditsPage__CreditListItem"
              >
                <p
                  class="Text Text__p"
                />
              </li>
              <li
                class="CreditsPage__CreditListItem"
              >
                <p
                  class="Text Text__p"
                >
                  Header/footer background ‘Argyle’ made by Will Monson from:
                  <button
                    class="Action Action__Appearance--a"
                    href="https://www.flaticon.com/authors/pixelmeetup"
                    title="Pixelmeetup"
                  >
                    https://www.transparenttextures.com
                  </button>
                </p>
              </li>
            </ul>
          </div>
        </div>
        <span
          style="display: table; clear: both;"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
